R/retrieve.R

```diff
@@ -1025,8 +1025,11 @@ renv_retrieve_successful <- function(record, path, install = TRUE) {
     renv_retrieve_handle_remotes(record, subdir = subdir)
 
   # ensure its dependencies are retrieved as well
-  if (state$recursive)
+  if (state$recursive) local({
```

Comment:
This could also be a place for us to turn on Additional_repositories?

Reply:
Yeah, that'd make sense.